language/tools/move-package/src/resolution/resolution_graph.rs

```diff
@@ -348,6 +357,24 @@ impl ResolvingGraph {
             );
         }
 
+        match dep.digest {
```

Comment:
What is this doing exactly? jsut need a bit more context

Reply:
In the package system you can specify a `digest` for a dependency, e.g., 
```
digest = "PcwYcMQt/UHh3YQ+7jDPlzhesvhRriZk0R+VhzrhCnT9ngcNB3Q8aTyXij1sE+pOsJpytr8kFeG6Sq/QPbPfuw=="
```
When we're compiling that dependency, we will then compute the digest of that package and compare the two. If they match resolution/compilation can continue. Otherwise we raise an error.

This is a way of essentially fixing your dependencies and making sure they cannot change or shift under your feet.